applications/API/models.py

```
@@ -0,0 +1,656 @@
+                verbose_name_plural = _("Themes")
+                managed = True
+                ordering = ["name"]
+                constraints = [
+                    models.UniqueConstraint(
+                        fields=["theme_key", "name"], name="unique_theme_key_name"
+                    ),
+                ]
+                indexes = [
+                    models.Index(fields=["theme_key"], name="idx_theme_key"),
+                    models.Index(fields=["name"], name="idx_theme_name"),
+                ]
```

Ruff (0.14.0)

Order of model's inner classes, methods, and fields does not follow the Django Style Guide: `Meta` class should come before Magic method

(DJ012)